### PR TITLE
Added UUID field to ProgramCertificate model

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-6481

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

**Comment by amangano-edx** [comment]
This seems to include a lot of extra stuff other than just the UUID change (translations files, etc). Would it make sense to break out some of this other stuff into separate PRs?

**Comment by clintonb** [comment]
@amangano-edx the cleanup work is done in a separate commit. I'm not convinced a separate PR would add much value; but, if you feel strongly, I can create one.

File: credentials/apps/credentials/tests/test_models.py [hunk 1]
```diff
@@ -143,3 +143,14 @@ def test_invalid_course_key(self):
             ).full_clean()
 
         self.assertEqual(context.exception.message_dict, {'course_id': ['Invalid course key.']})
+
+
+class ProgramCertificateTests(TestCase):
+    """ Tests for the ProgramCertificate model. """
+
+    def test_str(self):
```
Comment (amangano-edx) [inline credentials/apps/credentials/tests/test_models.py]: Would it make sense to add a docstring here to clarify what `str` is expected to return?

File: credentials/conf/locale/eo/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,319 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (amangano-edx) [inline credentials/conf/locale/eo/LC_MESSAGES/django.po]: Does this comment need to be updated?

File: credentials/conf/locale/eo/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-11-29 07:22+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
```
Comment (amangano-edx) [inline credentials/conf/locale/eo/LC_MESSAGES/django.po]: Does any of this need to be updated?

**Comment by amangano-edx** [comment]
@clintonb I don't feel strongly about it here, but in general I've found it helpful to keep PRs as small and focused as possible to minimize the risk of regressions slipping through and to make it easier for reviewers to catch them.

File: credentials/apps/credentials/tests/test_models.py (hunk 1 above)
Comment (clintonb) [inline credentials/apps/credentials/tests/test_models.py]: I don't think so. [`__str__()`](https://docs.python.org/2/reference/datamodel.html#object.__str__) is a pretty common built-in method.

**Pull request merged**